cmd/cli/main.go

```diff
@@ -4,16 +4,15 @@ import (
 	"octavius/internal/cli/command"
 	"octavius/internal/cli/config"
 	"octavius/internal/cli/daemon"
-	"octavius/internal/cli/fileUtil"
-	"octavius/internal/cli/logger"
-	"octavius/internal/cli/printer"
+	"octavius/internal/pkg/log"
 )
 
 func main() {
-	newPrinter := printer.GetPrinter()
-	logger.Setup()
-	fileUtil := fileUtil.NewFileUtil()
+	// TODO: get log level from config and define log file path if any
+	log.Init("info", "")
```

Comment:
Should we get the `logfile path` from the config as well? Can't we just define it inside the log initialiser depending on the case?

Reply:
Yes, that would be good because if we want to change the log path we don't need to do any deployment/version update. but we can provide a default path if the user not provide the path. The only thing we need to decide if this log needs to write to file or not.